crypto-ffi/src/bundles/welcome.rs

```diff
@@ -28,7 +28,7 @@ impl From<core_crypto::prelude::WelcomeBundle> for WelcomeBundle {
             crl_new_distribution_points,
         }: core_crypto::prelude::WelcomeBundle,
     ) -> Self {
-        let id = conversation_id_coerce_maybe_arc(id);
+        let id = conversation_id_coerce_maybe_arc(id.as_ref());
```

Comment:
We could _definitely_ adjust this implementation to avoid the need for manual call site adjustments for items in this commit.